### DISCUSSION TITLE
[Kavita] Smart filtrer for comics released this week (NCBD)

### DISCUSSION THREAD

**Discussion by MnsieurJF** [Kareadita/Kavita#3041] [Ideas]
### Idea Description

On the smart filter panel, could we have a criterion to get all comics that were released this week.
Example :
- release date of the last chapter is between today and today minus 7 days. 
- release date of the last chapter is in current week

In the comics' industry, there is this tradition called New Comic Book Day (NCBD). That is the day new comics from every publisher become available either on store or digitally. It's Wednesday every week.
I would be a great feature for the comics side of Kavita and every reader that reads comics on a weekly basis and follows the industry.

Also, legal platform like Marvel Unlimited, Omnibus and Comixology have a category "released this week".

It's also a way to keep track of recent addition to our library without being polluted by other older series being added.

Overall, could we have a more flexible date filter?

We discussed this [here ](https://discord.com/channels/821879810934439936/821879810934439939/1256204471399092244) on the discord channel and as requested I create this FR to keep track of it :)

Thank you

### Idea Category

Feature Enhancement

### Duration of Using Kavita

7 months

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by Cducharme84** [comment]
I like the idea of using last chapter release, I think you’d also appreciate the series level idea here too for more granular release filtering https://github.com/Kareadita/Kavita/discussions/2740
